scripts/main.sh

```diff
@@ -58,13 +60,25 @@ get_op_items() {
   #   }
   # ]
 
-  local -r JQ_FILTER="
-    .[]
-    | [select(.overview.URLs | map(select(.u == \"sudolikeaboss://local\")) | length == 1)?]
-    | map([ .overview.title, .uuid ]
-    | join(\",\"))
-    | .[]
-  "
+  if [ -n "$OPT_ITEMS_JQ_FILTER" ]; then
+    local -r JQ_FILTER="$OPT_ITEMS_JQ_FILTER"
+  elif [ "$OPT_ENABLED_URL_FILTER" == "on" ]; then
+    local -r JQ_FILTER="
+      .[]
+      | [select(.overview.URLs | map(select(.u == \"sudolikeaboss://local\")) | length == 1)?]
+      | map([ .overview.title, .uuid ]
+      | join(\",\"))
+      | .[]
+    "
+  else
+    local -r JQ_FILTER="
+      .[]
+      | [select(.overview.URLs | map(select(.u)) | length == 1)?]
+      | map([ .overview.title, .uuid ]
+      | join(\",\"))
+      | .[]
+    "
+  fi
```

Comment:
If you do like I suggested above, we could prevent this `if/else` chain and use the `OPT_ITEMS_JQ_FILTER` directly :D